datasets/schema.py

```diff
@@ -26,6 +26,12 @@ class DatasetSchemaDimensionCategoryNode(DjangoObjectType):
     class Meta:
         abstract = True
 
+
+class DatasetSchemaDimensionNode(DjangoObjectType):
+    class Meta:
+        abstract = True
```

Comment:
I don't know what this bit does but seems to be in line with everything else :sweat_smile: